creator-node/test/snapbackSM.test.js

```diff
@@ -810,4 +811,274 @@ describe('test SnapbackSM', function () {
     assert.ok(requiredUpdateReplicaSetOps[0].unhealthyReplicas.has('http://deregisteredCN.co'))
     assert.strictEqual(potentialSyncRequests[0].endpoint, 'http://cnWithSpId2.co')
   })
+
```

Comment:
unrelated nit but maybe we group tests by function inside describe blocks (can do in later PR)